publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -104,6 +109,23 @@ private void readSpringProperties() {
         }
     }
 
+    //it read and store the jasypt key from jasypt.key file
+    public static String readJasyptKeyFile(String jasyptPassword) {
+        String jasyptKey = "";
+        try {
+
+            if (StringUtils.isNotBlank(jasyptPassword)) {
+                jasyptKey = new String(Files.readAllBytes(Paths.get(jasyptPassword)));
+            } else {
+                return jasyptKey;
+            }
```

Comment:
remove this code, no use with these lines

Reply:
removed else part code